infrastructure/main.tf

```diff
@@ -59,31 +59,31 @@ resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
 resource "azurerm_key_vault_secret" "POSTGRES-USER-V11" {
   name         = "recipe-backend-POSTGRES-USER"
   value        = module.recipe-database-v11.user_name
-  key_vault_id = data.azurerm_key_vault.key_vault.id
+  key_vault_id = "${data.azurerm_key_vault.key_vault.id}-v11"
 }
 
 resource "azurerm_key_vault_secret" "POSTGRES-PASS-V11" {
   name         = "recipe-backend-POSTGRES-PASS"
   value        = module.recipe-database-v11.postgresql_password
-  key_vault_id = data.azurerm_key_vault.key_vault.id
+  key_vault_id = "${data.azurerm_key_vault.key_vault.id}-v11"
 }
 
 resource "azurerm_key_vault_secret" "POSTGRES_HOST-V11" {
   name         = "recipe-backend-POSTGRES-HOST"
   value        = module.recipe-database-v11.host_name
-  key_vault_id = data.azurerm_key_vault.key_vault.id
+  key_vault_id = "${data.azurerm_key_vault.key_vault.id}-v11"
 }
 
 resource "azurerm_key_vault_secret" "POSTGRES_PORT-V11" {
   name         = "recipe-backend-POSTGRES-PORT"
   value        = module.recipe-database-v11.postgresql_listen_port
-  key_vault_id = data.azurerm_key_vault.key_vault.id
+  key_vault_id = "${data.azurerm_key_vault.key_vault.id}-v11"
 }
 
 resource "azurerm_key_vault_secret" "POSTGRES_DATABASE-V11" {
   name         = "recipe-backend-POSTGRES-DATABASE"
   value        = module.recipe-database-v11.postgresql_database
-  key_vault_id = data.azurerm_key_vault.key_vault.id
+  key_vault_id = "${data.azurerm_key_vault.key_vault.id}-v11"
```

Comment:
what??? there is no key vault called this, and if one has been created it shouldn't have been

Reply:
looks like it exists which is breaking my pipeline because of duplicates. If it's not needed and not being used probably worth deleting it.

<img width="1120" alt="Azure_KV_Postgres-aat" src="https://user-images.githubusercontent.com/71495456/101799152-ba366600-3b03-11eb-8164-cfe9ee5855b9.png">

Reply:
@avi-hmcts what Tim meant is the id of the keyvault should not change. There is no keyvault id called which has "-v11"

Reply:
Yes my bad .. the issue was the duplicate names of the secrets.. corrected those now.